src/app/modules/browse/browse.tpl.html

```diff
@@ -44,7 +44,12 @@
                 <div class="container-fluid">
                     <div class="searchRow row">
                         <div class="col-md-6" style="padding-bottom:10px">
-                            <div ng-show="initialized">{{bookMessage}}</div>
+                            <div ng-show="initialized">
```

Comment:
Hm.  Worst case is we have a message saying there are no books simultaneous with the message allowing users to unhide custom ones.  That is less than ideal, however.  I will make the change.